### DISCUSSION TITLE
[Bug]  skywalking use in django and celery, the celery can not start well

### DISCUSSION THREAD

**Discussion by katelei6** [apache/skywalking#8711] [General]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Python Agent (apache/skywalking-python)

### What happened

I start celery, but not start success,then i ctrl + C three times, the celery can start.
![image](https://user-images.githubusercontent.com/79354895/158792156-8452a2e9-4874-4483-a14c-884a506f3d79.png)

1 time
![image](https://user-images.githubusercontent.com/79354895/158792220-6d8815ce-9576-404f-8462-cf48227839a2.png)
2 time
![Uploading image.png…]()
3 time 
![Uploading image.png…]()

### What you expected to happen

The right is when I start the celery ,it can run well.

### How to reproduce

The right is when I start the celery ,it can run well.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I assigned to ypu, and notice you have 2 screenshots not uploaded.

**Comment by katelei6** [comment]
1 time:
![image](https://user-images.githubusercontent.com/79354895/158793309-de347bb6-cbd3-4c71-9fb6-88238e371657.png)
2 time
![image](https://user-images.githubusercontent.com/79354895/158793352-30c520be-2dba-4c85-8770-bb7dd72bb88e.png)
3 time 
![image](https://user-images.githubusercontent.com/79354895/158793208-0ab19838-6e82-45e9-ac67-d742f0b039d2.png)

**Comment by Superskyyy** [comment]
![image](https://user-images.githubusercontent.com/26076517/159106676-54934f8b-e1a2-4064-9474-13ef828a910d.png)
@katelei6 Hello, thanks for reporting the issue, actually the current limitation was noted in the code and `plugins.md` documentation. 

Meanwhile, please try using the HTTP protocol and things should be working.

note = """The celery server running with "celery -A ..." should be run with the HTTP protocol
as it uses multiprocessing by default which is not compatible with the gRPC protocol implementation
in SkyWalking currently. Celery clients can use whatever protocol they want."""

It will need careful enhancements and I think there's a related issue pending for a while.  https://github.com/apache/skywalking/issues/7319

**Comment by Superskyyy** [comment]
If you can confirm Celery works fine after switching to HTTP, then I will move this one to discussions for now as it is documented and there's already an issue tracking the problem.
